src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,81 @@
+package duke.task;
+import duke.task.Task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+
+/**
+ * The deadline class is a task class that has an additional parameter by.
+ */
+public class Deadline extends Task {
+
+    protected LocalDateTime by;
+
+    /**
+     * Basic constructor.
+     * @param description The description of the task
+     * @param by The time the task should be completed by
+     */
+    public Deadline(String description, LocalDateTime by) {
+        super(description);
+        this.by = by;
+    }
+
+    /**
+     * Overloaded constructor where the isDone field can be set.
+     * @param description The description of the task.
+     * @param by The time the task should be completed by.
+     * @param isDone Whether the task has been completed.
+     */
+    public Deadline(String description, LocalDateTime by, boolean isDone) {
+        super(description,
```

Comment:
Any reason why you have the parameters on separate lines instead of a single line?

Reply:
Java coding standard for 2103 does not specify that params should be written in 1 single line. Even in examples given they are written in multiple lines.

Reply:
Agree with @yulonglim, but I also do acknowledge that it looks kind of awkward given such a short line